### DISCUSSION TITLE
How do i revert to the neovim default keybinds (and commands)

### DISCUSSION THREAD

**Discussion by coltonisgod234** [LazyVim/LazyVim#2322] [Q&A] [answered]
i need to revert to the *neovim defaults*, **NOT THE LAZYVIM ONES**. i don't like how the keybinds that lazyvim sets up are practicly forced on you, and don't get me wrong i LOVE lazyvim but the keybinds are making it a bit inaccessible to me.

edit: i'm a newbie to (neo)vim

**Comment by dpetka2001** [comment, marked as answer]
You can see the options you can pass to LazyVim spec [here](https://www.lazyvim.org/configuration#default-settings). To disable the keymaps you could do
```lua
-- ~/.config/nvim/lua/plugins/lazyvim.lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      defaults = {
        keymaps = false,
      },
    },
  },
}
```
Keep in mind this will only disable the keymaps defined in [lazyvim.config.keymaps](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/keymaps.lua). There might be also keymaps in the plugins' specs defined in the `keys` section, that you would have to disable manually.
